lab-andrew/test/brewer-router-test.js

```diff
@@ -0,0 +1,153 @@
+'use strict';
```

Comment:
These all look great, make sure to add a test for 404 errors on requests to bad endpoints as well, '/api/hello' for example.